src/interfaces/index.ts

```diff
@@ -68,22 +68,25 @@ export * from './utils-interfaces/validation-item-detail-interface'
 
 // External interfaces
 // NB: importing EmptyAddress doesn't work in local components
-export {
+export type {
```

Comment:
This is needed to the fix the "The requested module does not provide an export named" error.

Reply:
We probably need to do this in our other repos.

Reply:
OK to close this conversation?

Reply:
Yes, sounds good!